ENIGMAsystem/SHELL/Graphics_Systems/None/NONEStd.cpp

```diff
@@ -0,0 +1,95 @@
+/** Copyright (C) 2017 Faissal I. Bensefia
+*** Copyright (C) 2008-2013 Robert B. Colton, Adriano Tumminelli
+*** Copyright (C) 2014 Seth N. Hetu
+***
+*** This file is a part of the ENIGMA Development Environment.
+***
+*** ENIGMA is free software: you can redistribute it and/or modify it under the
+*** terms of the GNU General Public License as published by the Free Software
+*** Foundation, version 3 of the license or any later version.
+***
+*** This application and its source code is distributed AS-IS, WITHOUT ANY
+*** WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+*** FOR A PARTICULAR PURPOSE. See the GNU General Public License for more
+*** details.
+***
+*** You should have received a copy of the GNU General Public License along
+*** with this code. If not, see <http://www.gnu.org/licenses/>
+**/
+
+#include <iostream>
+#include <fstream>
+#include <string>
+#include <stdlib.h>     /* malloc, free, rand */
+#include <unordered_map>
+#include <math.h>
+
+using namespace std;
+#include "OPENGLStd.h"
+#include "Universal_System/shaderstruct.h"
+#include "Universal_System/scalar.h"
+#include "Universal_System/var4.h"
+#include "Universal_System/roomsystem.h" // Room dimensions.
+#include "Universal_System/depth_draw.h"
+#include "Universal_System/backgroundstruct.h"
+#include "Universal_System/image_formats.h"
+#include "Universal_System/spritestruct.h"
+#include "Universal_System/image_formats.h"
+#include "Universal_System/nlpo2.h"
+#include "Universal_System/fileio.h"
+#include "Universal_System/estring.h"
+#include "Collision_Systems/collision_types.h"
+#include "Universal_System/instance_system.h"
+#include "Universal_System/graphics_object.h"
+#include <algorithm>
+#include "Graphics_Systems/graphics_mandatory.h" // Room dimensions.
+#include "Graphics_Systems/General/GSprimitives.h"
+#include "../General/GSmath.h"
+#include "../General/GSbackground.h"
+#include "../General/GStextures.h"
+#include "../General/GStiles.h"
+#include "../General/GSvertex.h"
+#include "../General/GStextures.h"
+#include "../General/GSsurface.h"
+#include "../General/GSstdraw.h"
+#include "../General/GSsprite.h"
+#include "../General/GStextures.h"
+#include "../General/GLSurfaceStruct.h"
+#include "../General/GStextures.h"
+#include "../General/GSscreen.h"
+#include "../General/GSd3d.h"
+#include "../General/GSblend.h"
+#include "../General/GSmatrix.h"
+#include "../General/GScolors.h"
+#include "Platforms/platforms_mandatory.h"
+#include <floatcomp.h>
+#include <vector>
+#include <limits>
+#include <map>
+#include <list>
+#include <stack>
+#include "Universal_System/estring.h"
+#include "fillin.h"
+
+#define degtorad(x) x*(M_PI/180.0)
+unsigned get_texture(int texid){return -1;}
+
+using namespace enigma;
+using namespace enigma_user;
+
+namespace enigma {
+	unsigned char currentcolor[4] = {0,0,0,255};
```

Comment:
Is this definition being used by the code in `GScolors.cpp`? I suspect that the way it's using them can't be varied between systems, anyway, so we may as well move them there.

Reply:
All this stuff was the same between platforms so I moved it there, the rest was stuff that wasn't being used so I just straight up removed it.